spec/std/file_spec.cr

```diff
@@ -1160,8 +1160,8 @@ describe "File" do
       with_tempfile("utime-set.txt") do |path|
         File.write(path, "")
 
-        atime = Time.utc(2000, 1, 2)
-        mtime = Time.utc(2000, 3, 4)
+        atime = Time.utc(2000, 1, 2, nanosecond: 300_000_000)
+        mtime = Time.utc(2000, 3, 4, nanosecond: 500_000_000)
```

Comment:
Should I revert these two lines? This spec fails when subsecond precision isn't supported by the filesystem where the spec runs. Contemporary filesystems support subsecond precision, but old ones, such as FAT32, ext3 and HFS+, don't.

Reply:
reverted.